FabulousContacts/Controls/BorderedEntry.fs

```diff
@@ -7,27 +7,42 @@ open Xamarin.Forms
 type BorderedEntry() =
     inherit Entry()
 
-    static let BorderColorProperty = BindableProperty.Create("BorderColor", typeof<Color>, typeof<BorderedEntry>, Color.Default)
+    static let borderColorProperty =
+        BindableProperty.Create("BorderColor",
+                                typeof<Color>,
+                                typeof<BorderedEntry>,
+                                Color.Default)
 
     member this.BorderColor
-        with get () = this.GetValue(BorderColorProperty) :?> Color
-        and set (value) = this.SetValue(BorderColorProperty, value)
+        with get () =
+            this.GetValue(borderColorProperty) :?> Color
+        and set (value) =
+            this.SetValue(borderColorProperty, value)
 
 [<AutoOpen>]
 module DynamicViewsBorderedEntry =
-    let BorderedEntryBorderColorAttributeKey = AttributeKey<_> "BorderedEntry_BorderColor"    
+    let BorderedEntryBorderColorAttributeKey =
+        AttributeKey<_> "BorderedEntry_BorderColor"
 
     type Fabulous.XamarinForms.View with
-        static member BorderedEntry(?borderColor: Color,
-                                    ?placeholder, ?text, ?textChanged, ?keyboard) =
+        static member BorderedEntry(?borderColor: Color, ?placeholder, ?text, ?textChanged, ?keyboard) =
```

Comment:
To match the previous explanation, this line should be `static member inline`. I forgot to add it. :)